src/app/shared/interfaces/movies.ts

```diff
@@ -0,0 +1,25 @@
+
+class Results {
```

Comment:
it seems Results is your single movie, so it can be class Movie.

Reply:
you can also export this model and use it where you need instead of any

Reply:
1. Yes you're right, it's done.
2. I don't understand this completely, but we have other single movie interface winch is more complicated than Results.